src/components/Header/UserPreferences/Automation/Import.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactElement, useState } from 'react'
+import Button from '../../../@shared/atoms/Button'
+import { useAutomation } from '../../../../@context/Automation/AutomationProvider'
+import Loader from '../../../@shared/atoms/Loader'
+import styles from './Import.module.css'
+import Input from '../../../@shared/FormInput'
+import { LoggerInstance } from '@oceanprotocol/lib'
+
+export default function Import(): ReactElement {
+  const { isLoading, importAutomationWallet } = useAutomation()
+
+  const [showFileInput, setShowFileInput] = useState<boolean>()
+
+  const importToLocalStorage = async (target: EventTarget) => {
```

Comment:
nitpick: The function name importToLocalStorage might be misleading as the function doesn't seem to import anything to local storage (the called function might do it, but this is not the responsibility of this function). Consider renaming it to better reflect its purpose, like importWalletFromFile.